src/main/kotlin/ua/marchenko/artauction/common/annotation/scheduled/beanPostProcessor/CustomScheduledAnnotationBeanPostProcessor.kt

```diff
@@ -0,0 +1,35 @@
+package ua.marchenko.artauction.common.annotation.scheduled.beanPostProcessor
+
+import java.lang.reflect.Method
+import java.time.Duration
+import java.time.LocalDateTime
+import java.util.concurrent.Executors
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.TimeUnit
+import org.springframework.beans.factory.config.BeanPostProcessor
+import org.springframework.stereotype.Component
+import ua.marchenko.artauction.common.annotation.scheduled.annotation.CustomScheduled
+import ua.marchenko.artauction.common.annotation.scheduled.cron.DayTimeDetails
+
+@Component
+class CustomScheduledAnnotationBeanPostProcessor : BeanPostProcessor {
```

Comment:
What happens when application context shutdown? Will it continue to work some time?

Reply:
Yes, it would be working( I have added destroy() method in CustomScheduledInvokerContextListener to shutdown() the ScheduledExecutorService